web-app/js/portal/filter/ui/DateFilterPanel.js

```diff
@@ -101,7 +101,14 @@ Portal.filter.ui.DateFilterPanel = Ext.extend(Portal.filter.ui.BaseFilterPanel,
 
         this.filter.setValue({
             fromDate: this.fromDate.getValue(),
-            toDate: this.toDate.getValue()
+            toDate: this._endOfDay(this.toDate.getValue())
         });
+    },
+
+    _endOfDay: function(date) {
```

Comment:
I tried that function. The results were not really what is wanted as i dont think it handles the fact that we want the last moment of a UTC day not the last second of the date in client timezone expressed as UTC